plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryType.java

```diff
@@ -28,15 +30,15 @@ public void testTimeToStringConverter()
     {
         assertThat(BigQueryType.timeToStringConverter(
                 Long.valueOf(303497217825L)))
-                .isEqualTo("'16:00:00.148'");
+                .isEqualTo("'00:00:00.303497'");
```

Comment:
Not sure of the value of this test, but if it stays, add a test case with input that will result in something bigger than fraction of a second.